app/helpers/buyers_helpers.py

```diff
@@ -20,7 +20,8 @@ def get_framework_and_lot(framework_slug, lot_slug, data_api_client, status=None
 def is_brief_correct(brief, framework_slug, lot_slug, current_user_id):
     return brief['frameworkSlug'] == framework_slug and \
         brief['lotSlug'] == lot_slug and \
-        is_brief_associated_with_user(brief, current_user_id)
+        is_brief_associated_with_user(brief, current_user_id) and \
```

Comment:
If you wrap the entire expression in a pair of parenthesis then there's no need for escaping the newlines. This is the [advice PEP8 gives](https://www.python.org/dev/peps/pep-0008/#id19) (although it looks like the linter doesn't catch this):

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

```python
return (
     ...
     and ...
)
```